pkg/shell/manifest.json.in

```diff
@@ -28,5 +28,7 @@
             "environ": [ "SUDO_ASKPASS=@libexecdir@/cockpit-askpass" ],
             "spawn": [ "@SUDO@", "-A", "cockpit-bridge", "--privileged" ]
         }
-    ]
+    ],
+
+    "content-security-policy": "script-src 'self' 'unsafe-eval'"
```

Comment:
This is a security regression in the shell component. We need to find another way to accomplish this. Enabling use of eval breaks our security, and is the browser equivalent to partial ```setenforce 0```.